balena-haskell-hello-world.cabal

```diff
@@ -0,0 +1,26 @@
+Name:                balena-haskell-hello-world
+Version:             1.0
+License:             Apache-2.0
+Author:              vipulgupta2048
+Maintainer:          vipul@balena.io
+Stability:           Experimental
+Category:            Web
+Build-type:          Simple
+Cabal-version:       >=1.2
```

Comment:
That's super old. Are you sure it works with 1.2, and do we need that much backwards compatibility?

Reply:
It was generated by `snap`  from the boilerplate. Is there a way to generate dependencies or the stack.yml file using stack there must be right. Hopefully I can replace cabal with Stack